src/model/Clearing.java

```diff
@@ -3,19 +3,23 @@
 import java.util.Arrays;
 
 public class Clearing {
-	
+
     public String type;
     public Garrison dwelling = null; 			// not sure what type this should actually be
     public int location;		 		// indicating which clearing on the tile this is CHANGED BACK TO AN INT
     public Clearing[] connections;
     public int nextConnection =0;
     public Tile parent;
-    
+    public Treasure[] treasures;
+
+
     Clearing(int location, Tile parent) {
         this.type     = "";
         this.location = location;
-        this.parent = parent;
-        connections = new Clearing[4];		//4 is the most connections had by any 
+        this.parent   = parent;
+        connections   = new Clearing[4];		//4 is the most connections had by any
+        treasures     = new Treasure[10]; // arbitrary number
+        // TODO: how should the treasures be added to the tile?
```

Comment:
I had originally planned to do treasures really nicely this iteration but
we will see if there is time.
The boardsetup code has some comments regarding treasure placement but I am
not sure off hand how clear they are.

On Sun, Feb 22, 2015 at 1:28 PM, ChantalForget <notifications@github.com>
wrote:

> In src/model/Clearing.java
> <https://github.com/sstone93/MAGIC/pull/38#discussion_r25134912>:
>
> >      Clearing(int location, Tile parent) {
> >          this.type     = "";
> >          this.location = location;
> > -        this.parent = parent;
> > -        connections = new Clearing[4];		//4 is the most connections had by any
> > +        this.parent   = parent;
> > +        connections   = new Clearing[4];		//4 is the most connections had by any
> > +        treasures     = new Treasure[10]; // arbitrary number
> > +        // TODO: how should the treasures be added to the tile?
>
> @SunsetBlade <https://github.com/SunsetBlade> I wasn't sure if the board
> would place the treasures around when its setup, or what
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sstone93/MAGIC/pull/38/files#r25134912>.
>



-- 
Nicholas Laws